docusaurus/docs/code-base-works/directory-structure.md

```diff
@@ -27,6 +27,14 @@ Components in the dialog folder are used within the `PromptModal` component. Dis
 ## formatters 
 This is not a top-level folder in the shell, which uses `/components/formatter`, but a top-level `formatters` directory works the same way in an extension as the shell `formatter` directory does. Formatters are used to format data within tables. see [Defining Products](./products-and-navigation.md) for more information on configuring resource tables.
 
+## layouts
+Components in this folder are used as layouts in other components. This is an artifact from our Nuxt usage and more details can be read about that in the Nuxt [documentation](https://nuxt.com/docs/guide/directory-structure/layouts).
+
+The existing layouts available can be found [here](https://github.com/rancher/dashboard/tree/master/shell/list).
+
+### Overlapping Layout Names
+If an extension uses the same name as an existing layout the extension's layout will override the shell's layout. This decision may get revisited in the future. 
```

Comment:
I would say here that we discourage the usage of overlapping layout names because of the side effects (bug fixes on a dashboard layout you are overwriting will not be there, causing potencial side effects on the dashboard instance the user is running). If you agree with me, let's get this in.

Otherwise, let me know your opinion mate.

Reply:
Updated the comment to discourage usage.